resume.md

```diff
@@ -1,49 +1,34 @@
-# FIRST LAST NAME
+# Niharika Desaraju 
 
 (123) 456 - 7890 | email@gmail.com
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+github.com/ndesaraju | linkedin.com/in/niharika-desaraju
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**University of California, Berkeley** | B.S. Bioengineering, Minor Computer Science May 2021
+* **Coursework**: Data Structures, Algorithms, Machine Learning, Discrete Math, Probability, Computational Biology, Databases, Computer Architecture, Computer Security.
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
+* **Languages**: Python, Java, C, SQL, Swift
+* **Libraries/Frameworks**: Pandas, TensorFlow, RESTful APIs, Node.JS, Flask, Numpy, HealthKit, ResearchKit
+* **Version Control**: Git
 * *...*
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Research Intern**: UCSF - Henry Roland Laboratory | August 2018-Present
+* ● Built a secure Apache HTTPS Server to create a dynamic web application for physicians and researchers to query for and visualize volumetric data.
```

Comment:
How did these queries matter? What business metrics did this drive? How did your contributions make work better for your clients?